test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.common;
+
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+public class UtilsTest {
+	ArrayList<String> test = new ArrayList<String>();
+	
+	@Test
+	public void isAnyNull_expectedNull() {
+		assertTrue(Utils.isAnyNull("1", null));
+	}
+	
+	@Test
+	public void isAnyNull_expectedNotNull() {
```

Comment:
the test name should have 3 parts `<method being tested>_<what is being tested>_<what is the expected return value>`
you can eliminate part 3 and 2 only if you have broadened the scope of the test to include testing multiple things.